recipe/meta.yaml

```diff
@@ -35,21 +42,55 @@ requirements:
     # need a CDT for libudev for cos6, proper package available for cos7 (below)
     - {{ cdt('libudev-devel') }}  # [linux and cdt_name=='cos6']
   host:
+    - python
     - libudev  # [linux and cdt_name!='cos6']
     - libusb
   run:
+    - python
```

Comment:
Why we add back the run dependency on python? We explicitly remove it with ignore_run_exports

Reply:
I think that was left over from an older revision, I've removed it now